pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -50,6 +50,18 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 		return nil
```

Comment:
nit: the new check should live before this block that's reading out the `OperatorUpgradeable` condition.